platform/broadcom/docker-saiserver-brcm.mk

```diff
@@ -2,7 +2,7 @@
 
 DOCKER_SAISERVER_BRCM = docker-saiserver$(SAITHRIFT_VER)-brcm.gz
 $(DOCKER_SAISERVER_BRCM)_PATH = $(PLATFORM_PATH)/docker-saiserver-brcm
-$(DOCKER_SAISERVER_BRCM)_DEPENDS += $(SAISERVER)
+$(DOCKER_SAISERVER_BRCM)_DEPENDS += $(SAISERVER) $(SAISERVER_DBG)
```

Comment:
Is it correct? Normally, we only install debug packages in debug dockers.

Reply:
yes, this docker is only used for testing, in order to make it easier to debug, make it always go with the debug package